src/com/projectkorra/projectkorra/GeneralMethods.java

```diff
@@ -1722,6 +1723,15 @@ public static boolean isRegionProtectedFromBuildPostCache(final Player player, f
 				final Land land = Land.getLand(location);
 				if (land != null) {
 					final Kingdom kingdom = land.getKingdom();
+					if (kPlayer.isAdmin()) {
+						return false;
+					}
+					if (land.getInvasion() != null) {
+						final Invasion invasion = land.getInvasion();
+						if (invasion.getInvader().equals(kPlayer) && invasion.getDefender().equals(land)) {

```

Comment:
You may want to make this a config option. Some people may not want invaders to be able to bend in the land they're i nvading

Reply:
Ok, I'll add a config option.